### PR TITLE
Master

### PR DESCRIPTION
**Pull request by MistaFish31**

### CODE REVIEW

**Comment by heyryanleys** [comment]
@MistaFish31 If you move your text from functionDraw to functionSetup, you won't get the distorted text effect that can show up sometimes. Here's what it looks like in draw vs setup:

Draw:
screen shot 2016-03-05 at 4 42 19 pm

Setup:
screen shot 2016-03-05 at 4 42 09 pm

So your code would look like:

 `function setup() {
  createCanvas(1100,700)
  noStroke()
  
  var spacing=80
  for (entry in IQ_DATA.data) {
    var IQ_pointHeight=height-(IQ_DATA.data[entry]*1.5+60);
    spacing+=11
    
    fill('#66f')
    ellipse(spacing, IQ_pointHeight, 8, 8)
  }
 
  var spacing=80
  for (entry in LIT_DATA.data) {
    var LIT_pointHeight=height-(LIT_DATA.data[entry]*1.5+240);
    spacing+=11
    
    fill('#f66')
    ellipse(spacing, LIT_pointHeight, 8, 8)
  }
//TITLE
  noStroke()
  textSize(26)
  fill('#555')
  text("Do countries with low literacy rates also show disproportionately low IQ averages?",75,85,600,200)
  
//KEY
  textSize(17)
  fill('#66f')
  text("IQ Level",90,180,500,200)
  ellipse(75,190,8,8)
  fill('#f66')
  text("Literacy Rate",90,205,500,200)
  ellipse(75,215,8,8)
  
//GRAPH
  noFill()
  stroke('#555')
  rect(70,250,950,400)
}`

**Comment by MistaFish31** [comment]
Thank you Ryan! Sorry it took me a while to reply; I appreciate the help a lot though and I'll apply this to my code soon :)

**Comment by MistaFish31** [comment]
screen shot 2016-04-03 at 6 42 04 pm

File: sketch.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  STAT_DIGITS = 2;
+  STAT_PREFIX = '';
+  STAT_POSTFIX = '%';
+  
+function setup() {
+  createCanvas(1100,700)
+  noStroke()
+  
+  var spacing=80
```
Comment (FranDias) [inline sketch.js]: You only need to declare `var spacing` once then can just reset the value.

File: sketch.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  for (entry in IQ_DATA.data) {
+    var IQ_pointHeight=height-(IQ_DATA.data[entry]*1.5+60);
+    spacing+=11
+    
+    fill('#66f')
+    ellipse(spacing, IQ_pointHeight, 8, 8)
+  }
+ 
+  var spacing=80
```
Comment (FranDias) [inline sketch.js]: Don't need `var` here`

File: sketch.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  for (entry in LIT_DATA.data) {
```
Comment (FranDias) [inline sketch.js]: Nice use of `for in` 👍

**Pull request closed without merging**